06_modules_lab/01.py

```diff
@@ -1,4 +1,12 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times
-"""
+'''
+Prints 'Hello Python' message x times when x is received as a parameter to the script
+Prints usage if
+'''
+import sys
+
+if len(sys.argv) == 2 and sys.argv[1].isdigit():
+    arg = int(sys.argv[1])
+    for _ in range(0, arg):
+        print 'Hello Python'
+else:
+    print 'Invalid usage. Usage: python 01.py <Number>'

```

Comment:
What if a user changed the file name? A more robust solution is to use `sys.argv[0]` as the program name